internal/context/apply.go

```diff
@@ -0,0 +1,65 @@
+package context
+
+import (
+	"fmt"
+	"log"
+	"path"
+
+	"github.com/commitdev/zero/internal/config/projectconfig"
+	"github.com/commitdev/zero/pkg/util/exit"
+	"github.com/manifoldco/promptui"
+)
+
+// Apply will load the context/configuration to be used by the apply command
+func Apply(applyEnvironments []string, applyConfigPath string) *projectconfig.ZeroProjectConfig {
+	if len(applyEnvironments) == 0 {
+		fmt.Println(`Choose the environments to apply. This will create infrastructure, CI pipelines, etc.
+At this point, real things will be generated that may cost money!
+Only a single environment may be suitable for an initial test, but for a real system we suggest setting up both staging and production environments.`)
+		applyEnvironments = promptEnvironments()
```

Comment:
This won't affect the value of `applyEnvironments` in the calling code, but it might be better to merge this and the stuff in the other file into the same package anyway.